authz/providers/azure/rbac/rbac.go

```diff
@@ -443,33 +449,50 @@ func (a *AccessInfo) CheckAccess(request *authzv1.SubjectAccessReviewSpec) (*aut
 	}
 
 	// Fallback to managed namespace check
-	if !a.useManagedNamespaceResourceScopeFormat || a.clusterType != managedClusters {
-		klog.V(7).Infof("Checkaccess request is denied for user %s", checkAccessUsername)
-		return status, nil
-	}
+	managedNamespaceExists, managedNamespacePath := getManagedNameSpaceScope(request)
```

Comment:
I assume there is no behavior change to access check for managed namespace of single cluster.